Jenkinsfile

```
@@ -30,6 +30,31 @@ boolean doc_only_change() {
     return rc == 1
 }
 
+boolean release_candidate() {
+    sh(label: "Determine if building (a PR of) an RC",
+           script: "git diff-index --name-only HEAD^ | grep -q TAG && " +
+                   "grep -i '[0-9]rc[0-9]' TAG",
+          returnStatus: true)
```

Have you confirmed that a function exits with the status of the last function that it called or do you actually need:

```suggestion
    return sh(label: "Determine if building (a PR of) an RC",
              script: "git diff-index --name-only HEAD^ | grep -q TAG && " +
                      "grep -i '[0-9]rc[0-9]' TAG",
             returnStatus: true)
```

In any case, your indentation needs fixing even if you don't need the `return`:

```suggestion
    sh(label: "Determine if building (a PR of) an RC",
       script: "git diff-index --name-only HEAD^ | grep -q TAG && " +
               "grep -i '[0-9]rc[0-9]' TAG",
       returnStatus: true)
```